querybook/webapp/components/StatementResultTable/StatementResultTable.tsx

```diff
@@ -143,43 +172,63 @@ export const StatementResultTable: React.FunctionComponent<{
     const showPagination = rows.length > maxNumberOfRowsToShow && paginate;
 
     return (
-        <StyledTableWrapper fontSize={tableFontSize}>
-            <Table
-                minRows={0}
-                stickyHeader
-                className="-highlight force-scrollbar-x"
-                defaultPageSize={
-                    paginate
-                        ? Math.min(maxNumberOfRowsToShow, rows.length)
-                        : null
-                }
-                showAllRows={!paginate}
-                rows={filteredRows}
-                cols={columns}
-                showPagination={showPagination}
-                formatCell={(index, column, row) => {
-                    const transformer = getTransformerForColumn(index);
-                    return transformer
-                        ? transformer.transform(row[index])
-                        : row[index];
-                }}
-                sortCell={useSortCell(rows)}
-            />
-        </StyledTableWrapper>
+        <>
+            <ShowFullContentCellSwitch>
+                <FullContentText>
+                    {isDisplayFull ? 'Hide' : 'Show'} full cell content:
+                </FullContentText>
+                <ToggleSwitch
+                    checked={isDisplayFull}
+                    onChange={visibilityToggle}
+                />
+            </ShowFullContentCellSwitch>
+            <StyledTableWrapper fontSize={tableFontSize}>
+                <Table
+                    minRows={0}
+                    stickyHeader
+                    className="-highlight force-scrollbar-x"
+                    defaultPageSize={
+                        paginate
+                            ? Math.min(maxNumberOfRowsToShow, rows.length)
+                            : null
+                    }
+                    showAllRows={!paginate}
+                    rows={filteredRows}
+                    cols={columns}
+                    showPagination={showPagination}
+                    formatCell={(index, column, row) => {
+                        const transformer = getTransformerForColumn(index);
+                        return transformer
+                            ? transformer.transform(row[index])
+                            : row[index];
+                    }}
+                    sortCell={useSortCell(rows)}
+                />
+            </StyledTableWrapper>
+        </>
     );
 };
 
+function useSavePrevValue() {
```

Comment:
this hook is not needed, but for future reference there is a helper hook called `usePrevious` that does the same thing